### PR TITLE
simplify the homepage

### LINKED ISSUE #153: As a user, I want to know where I can go to get help
- If 18F staff, use #cloud-gov-support.
- Otherwise, use [the Google Group](https://groups.google.com/a/gsa.gov/forum/#!overview).
- If there's an issue with the documentation, [open an issue](https://github.com/18F/cg-docs/issues/new).

@mogul Does that sound right, for the current state of things?

### PR DESCRIPTION
**Pull request by afeld**
Inspired by https://github.com/18F/cg-landing/pull/37#issuecomment-168226981, this pull request encompasses a few things:
- Create a dedicated Help page, and expand on the different support channels – closes #153
- Remove the Help and Public Domain sections from the homepage, since those won't be applicable to first-time visitors
- Add a description of the different documentation sections on the homepage
- Add a CC0 badge to the footer

screen shot 2015-12-31 at 3 45 07 pm

/cc @jameshupp @brethauer

### CODE REVIEW

File: content/help.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Talk to us in [#cloud-gov-support](https://18f.slack.com/messages/cloud-gov-support).
+
+## I found a problem with the documentation.
+
+[Open an issue on GitHub](https://github.com/18F/cg-docs/issues/new).
+
+## I'm interested in working on cloud.gov.
+
+Check out [18F's jobs site](https://pages.18f.gov/joining-18f/), and come talk to us in [our #devops-public Slack channel](https://chat.18f.gov/).
```
Comment (afeld) [inline content/help.md:25]: @mogul Does all of the information here seem accurate, for the current state of things? Any sorts of requests you can think of that I didn't mention? Is there too much?

File: content/index.md [hunk 2]
```diff
@@ -6,14 +6,18 @@ type: index
...
-> This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
->
-> All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
+### Usage
+
+Learn how to deploy applications to cloud.gov.
+
+### Contributing
+
+Intended for cloud.gov operations staff, but made public for others to learn how we run the platform.
```
Comment (afeld) [inline content/index.md:23]: Don't love my various descriptions here...curious if others can think of better wording, or if they are helpful at all.

File: content/ops/repos.md [hunk 3]
```diff
@@ -27,3 +27,11 @@ Here are the repos with the tools and configuration for the platform:
...
 
 We try and keep the page up-to-date, but you can find more repositories [on GitHub](https://github.com/search?o=desc&q=user%3A18F+%28cf+OR+cg+OR+%22cloud+foundry%22+OR+cloud.gov%29+NOT+cfn&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93).
+
+## Public domain
+
+This project is in the worldwide [public domain](https://github.com/18F/cloud-foundry-notes/blob/master/LICENSE.md). As stated in [CONTRIBUTING](https://github.com/18F/cloud-foundry-notes/blob/master/CONTRIBUTING.md):
+
+> This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+>
+> All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
```
Comment (afeld) [inline content/ops/repos.md:37]: This was a somewhat arbitrary choice of where to put this. Open to other ideas.

File: content/help.md (hunk 1 above)
Comment (dlapiduz) [inline content/help.md:25]: I wonder if we should have 18F specific stuff in our docs. What do you think @mogul?

Comment (mogul) [inline content/help.md:25]: I kind of agree, but let's make purging it a specific endeavor.

**Pull request merged**